src/main/app/claims/models/countyCourtJudgment.ts

```diff
@@ -0,0 +1,43 @@
+import { TheirDetails } from 'claims/models/details/theirs/theirDetails'
+import { PaymentType } from 'ccj/form/models/ccjPaymentOption'
+import { RepaymentPlan } from 'claims/models/replaymentPlan'
+
+abstract class CountyCourtJudgment {
+
+  constructor (public defendant: TheirDetails,
+               public paymentOption: string,
+               public paidAmount: number) {
+
+    this.defendant = defendant
+    this.paymentOption = paymentOption
+    this.paidAmount = paidAmount
+  }
+}
+
+export class CountyCourtJudgmentImmediatePayment extends CountyCourtJudgment {
```

Comment:
I don't think we want to use inheritance here, it looks overly complex

Reply:
you might be right